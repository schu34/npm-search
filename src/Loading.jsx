import React from "react";
import PropTypes from "prop-types";
import { Color } from "ink";

const Loading = ({ message }) => <Color blue> {message} </Color>;

Loading.propTypes = {
  message: PropTypes.string.isRequired
};

export default Loading;
