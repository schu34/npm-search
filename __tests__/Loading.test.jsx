import React from "react";
import Loading from "../src/Loading";
import { render } from "ink-testing-library";

describe("<Loading />", () => {
  it("should render without errors", () => {
    const props = {
      message: "test test",
    };
    const { lastFrame } = render(<Loading {...props} />);
    expect(lastFrame()).toMatch("test test");
  });
});
