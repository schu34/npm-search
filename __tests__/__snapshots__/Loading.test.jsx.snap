// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render without errors 1`] = `"[34mtest test[39m"`;
